<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多宿舍管理系统</string>
  </property>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>151</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>串口号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Cboxport"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Cboxboudrate">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>校验位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Cboxparity">
        <item>
         <property name="text">
          <string>NONE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ODD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EVEN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MARK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SPACE</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Cboxdatabits">
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>停止位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Cboxstopbits">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>95</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btn_open">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTextEdit" name="textEdit_Recv">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_send">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>接收区：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>发送区：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>下位机控制：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_yellow">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_red">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>黄色LED</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>红色LED</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>520</y>
     <width>147</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>蜂鸣器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_bee">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>361</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>教室编号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>室内温度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_Temp"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>室内湿度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>工作模式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>室内人数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btn_yellow</tabstop>
  <tabstop>Cboxport</tabstop>
  <tabstop>Cboxboudrate</tabstop>
  <tabstop>Cboxparity</tabstop>
  <tabstop>Cboxdatabits</tabstop>
  <tabstop>Cboxstopbits</tabstop>
  <tabstop>btn_open</tabstop>
  <tabstop>textEdit_Recv</tabstop>
  <tabstop>btn_red</tabstop>
  <tabstop>btn_bee</tabstop>
  <tabstop>btn_send</tabstop>
  <tabstop>textEdit_send</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
